lib/tasks/after_tasks.rake

```diff
@@ -267,5 +267,30 @@ namespace :After do
     r&.destroy
   end
 
+  desc "Remove full-width and ideographic commas from tags"
+  task(remove_invalid_commas_from_tags: :environment) do
+    puts("Tags can only be renamed by an admin. Enter your admin login:")
+    login = $stdin.gets.chomp.strip
+    admin = Admin.find_by(login: login)
```

Comment:
Would it be worthwhile to check the password too, with `admin.valid_password?(...)`?

Reply:
tbh I'm not really doing it to make sure that the admin is authorized, just to avoid the validation failing (I was actually thinking about using a wrangling chair's admin account to avoid confusion about why a non-wrangling admin was renaming tags)